schemas/oic.sec.cred.json

```diff
@@ -62,7 +62,7 @@
           }
         }
       },
-      "required": [ "credid", "subjectuuid", "credtype" ]
```

Comment:
I believe credid should be required... let's resolve in email before committing this PR